src/csvcubed/models/cube/qb/components/codelist.py

```diff
@@ -91,6 +112,14 @@ def __post_init__(self):
             self.concept_scheme_uri = None  # type: ignore
             self.concept_template_uri = None  # type: ignore
 
+    def _get_validations(self) -> Dict[str, ValidationFunction]:
```

Comment:
The `_csvw_contains_sufficient_information_validator` function doesn't have an equivalent validation function in the new `ValidatedModel` world.

Reply:
Just checking to be sure, does this mean we need to write a new function to validate _csvw_contains_sufficient_information_validator?

Reply:
It meant a bit more than that. We have to add some functionality to support whole-object validations.

Reply:
Added an equivalent function that does whole-object validation, definitely needs checking though.